src/app/shared/services/search.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import { Movies } from '@shared/interfaces/movies';
+import { APIURL } from '@shared/apiUrl';
+import { Observable, of } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { HttpClient } from '@angular/common/http';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class SearchService {
+
+  // constructor(private http: HttpClient) { }
+  // private apiUrl: Object = APIURL;
```

Comment:
it is nice to name private members with _. When you see it in the middle of the file, you will know it is a private one.

Reply:
Done